extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -1,7 +1,12 @@
 require([
 	'jquery',
 	'wikia.tracker',
-], function ($, tracker) {
+	'wikia.ui.factory',
+	'wikia.mustache',
+	'wikia.window',
+	'embeddablediscussions.templates.mustache',
+	'EmbeddableDiscussionsSharing'
+], function ($, tracker, uiFactory, mustache, window, templates, sharing) {
 	'use strict';
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AStrictMode)